src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const fixedRobot = {};
+
+  for (const key in robot) {
+    if (fixedRobot.hasOwnProperty(key)
```

Comment:
as for me... redundant
```suggestion
    if (
```